tests/ft_rust.vader

```diff
@@ -119,35 +119,35 @@ Execute (cargo warning message):
   AssertEqual [7], g:lengths
   bwipe
 
-Execute (cargo error message children):
-  "Monkey patch to check setting of length
-  Save g:lengths
-  let g:lengths = []
-  function! neomake#highlights#AddHighlight(entry, type) abort
-      call add(g:lengths, a:entry.length)
-  endfunction
-
-
-  new
-  file tests/from.rs
-  let cargo = neomake#makers#ft#rust#cargo()
-  let cargo.exe = 'cat'
-  let cargo.args = 'tests/fixtures/rust/cargo_error_children.json'
-
-  call neomake#Make(1, [cargo])
-  NeomakeTestsWaitForFinishedJobs
-
-
-  AssertEqual [{
-    \ 'type': 'E',
-    \ 'bufnr': bufnr('%'),
-    \ 'nr': -1,
-    \ 'lnum': 11,
-    \ 'col': 10,
-    \ 'valid': 1,
-    \ 'vcol': 0,
-    \ 'pattern': '',
-    \ 'text': 'custom derive attribute panicked. message: Only structs and enums can derive From'}], getloclist(0)
-
-  AssertEqual [4], g:lengths
-  bwipe
+" Execute (cargo error message children):
+"   "Monkey patch to check setting of length
+"   Save g:lengths
+"   let g:lengths = []
+"   function! neomake#highlights#AddHighlight(entry, type) abort
```

Comment:
I commented the monkey patch and that indeed fixes the issue. I just change it to not monkey patch in just nvim. If that works it's fine by me if you merge it like that. Although if you can think of a nicer solution that's obviously better.